### PR TITLE
Update User Guide

### PR DESCRIPTION
**Pull request by Lessthanfree**
Add command summary
Clean up all references to AB-4

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -12,19 +12,24 @@ ifdef::env-github[]
...
+:repoURL: https://github.com/CS2103-AY1819S1-T09-2/Main.git
 
-By: `T09-02`      Since: `Sep 2018`      Licence: `MIT`
+By: `Team T09-2`      Since: `Aug 2018`      Licence: `MIT`
 
 == Introduction
+Welcome to the *Anakin* User Guide!
 
-*Anakin*  is for those who *want to make heavy use of flashcards without the hassle of managing physical copies*. More importantly, Anakin is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+*Anakin*  was built for those who *want to make heavy use of flashcards without the hassle of managing physical copies*. More importantly, Anakin is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI).
```
Comment (yujiatay) [inline docs/UserGuide.adoc]: I think we can add in a few lines talking about `Purpose of User Guide` (CS2101 requirements). 

Example: 
>  This user guide provides in-depth documentation on the Anakin installation process, features...

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -64,15 +68,16 @@ Format: `list`
...
-Adds a person to the address book +
-Format: `newdeck [n/name]`
+Adds a new deck to Anakin +
+Format: `newdeck [n/NAME]`
 
 ****
-* Name will be taken as a string.
-* If name is left blank, the name of the deck will be “New Deck #number”.
+*If NAME is left blank, the name of the deck will be “New Deck #number”.
+*NAME will be taken as a string.
```
Comment (yujiatay) [inline docs/UserGuide.adoc]: Regression issues. Proper formatting was reverted.

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -209,6 +213,30 @@ Format: `exit`
...
+|`delcard INDEX_OF_DECK INDEX_OF_CARD` | Deletes the card at the specified INDEX_OF_CARD inside the deck at INDEX_OF_DECK.
+|`delcard INDEX_OF_CARD` | Deletes the card at the specified INDEX. +
+You must be inside a deck to perform this operation.
+|`newcard [q/QUESTION] [a/ANSWER]` | Creates a new card in the current deck. q/ and a/ are optional.
+|`editcard INDEX_OF_CARD [q/QUESTION] [a/ANSWER]` | Edits a new card in the current  at INDEX_OF_CARD. Either q/ or a/ must be provided.
+|`sortname` | Sort the currently displayed list of decks or cards by name
+|`classify RATING` | Classifies the current card into one of the following categories: {‘easy’, ‘good’ , ‘hard’ , ‘review’}
+
+
+|=======================================================================
```
Comment (yujiatay) [inline docs/UserGuide.adoc]: Nice work!

File: src/main/java/seedu/address/logic/Anakin_commands/AnakinListCommand.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.model.Anakin_Model;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class AnakinListCommand extends Anakin_Command {
```
Comment (yujiatay) [inline src/main/java/seedu/address/logic/Anakin_commands/AnakinListCommand.java]: Should not be in this pull req

Comment (Lessthanfree) [reply src/main/java/seedu/address/logic/Anakin_commands/AnakinListCommand.java]: Oh yeah this change got squashed into this PR accidentally.

**Pull request merged**